mise.toml

```
@@ -0,0 +1,16 @@
+[tasks.lint-setup]
+description = "Install linting tools from version-pinned files (managed by Dependabot)"
+run = """
+pip install -r .github/requirements.txt
+pnpm i --frozen-lockfile --dir ./.github
+"""
+
+[tasks.lint]
+depends = ["setup"]
```

```suggestion
depends = ["lint-setup"]
```

If you want to keep the dependency.

---

I'd rather keep it, at least for now, as it reduces friction in using the tool. and, at least on my machine, it takes 300ms altogether.